server.py

```diff
@@ -108,167 +155,178 @@ def json_serial(obj):
 
 
 def json_response(content):
-    output = json.dumps(content, default=json_serial)
+    output = json.dumps(object_as_dict(content), default=json_serial)
     return Response(output, mimetype='application/json')
 
 
-def save_response(bound_logger, survey_response):
-    doc = {}
-    doc['survey_response'] = survey_response
-    doc['added_date'] = datetime.utcnow()
+def object_as_dict(obj):
+    return {c.key: getattr(obj, c.key)
+            for c in inspect(obj).mapper.column_attrs}
+
+
+def queue_cs_notification(logger, tx_id):
+    publisher = QueuePublisher(logger, settings.RABBIT_URLS, settings.RABBIT_CS_QUEUE)
+    return publisher.publish_message(tx_id)
+
 
-    invalid_flag = False
+def queue_ctp_notification(logger, tx_id):
+    publisher = QueuePublisher(logger, settings.RABBIT_URLS, settings.RABBIT_CTP_QUEUE)
+    return publisher.publish_message(tx_id)
 
-    if 'invalid' in survey_response:
-        invalid_flag = survey_response['invalid']
+
+def queue_cora_notification(logger, tx_id):
+    publisher = QueuePublisher(logger, settings.RABBIT_URLS, settings.RABBIT_CORA_QUEUE)
+    return publisher.publish_message(tx_id)
+
```

Comment:
This is technical debt from previous commits (I've just moved it). You are right though - I'll update it.